hack/pin-bundle-images.sh

```diff
@@ -89,4 +92,5 @@ done
 if [ -z "$DOCKERFILE" ]; then
     # pin rabbit to sha256 for our v2.9.0_patches fork
     echo -n ",quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle@sha256:bad31e1b028840ac3c199efc3a8ecb3c87bdb68e6a11c8e32a7dbcd5a4d4114d"
+    echo -n ",quay.io/fpantano/nova-operator-bundle@sha256:b300194c5f51cd36fcb29d5a823f98f572e1eb364fbf711470d17a13f16640f7"
```

Comment:
I need this to tmp build the operator and not fail on pin-bundle-images jq loop.